### DISCUSSION TITLE
Host configuration

### DISCUSSION THREAD

**Discussion by kn0ledge** [petretiandrea/home-assistant-tapo-p100#297] [Q&A]
Hello everyone,

i managed to install the integration, but i don't know what to put in the field Host. 
Can anyone help me?

Thanks in advance

**Comment by peat13** [comment]
Hi, I had the same problem. You need to put the IP of your Tapo device, just open the app and select the device, then look in the Device Info. You need to add each device separately.

**Comment by ibennani** [comment]
You also have to set a static ip for each device. This is done in your router.
This is not mandatory,, but if your device change IP, you'll have to re-install the integration. Using static IP prevents this from happening.
